nl_NL/LC_MESSAGES/tickets.po

```diff
@@ -206,37 +212,38 @@ msgid "TICKET_DESC_neighborhood_watch_ticket"
 msgstr "Bedankt dat je helpt Sudomemo een veilige en gastvrije plek te houden!"
 
 msgid "TICKET_NAME_featured_flipnote_ticket"
-msgstr "Featured Flipnote Ticket"
+msgstr "uitgelichte Flipnote Ticket"
 
 msgid "TICKET_DESC_featured_flipnote_ticket"
-msgstr "Congratulations! One of your Flipnotes was featured."
+msgstr "gefeliciteerd! je flipnote is uitgelicht!"
 
 msgid "TICKET_NAME_featured_artist_ticket"
-msgstr "Featured Artist Ticket"
+msgstr "Uitgelichte artiest ticket"
 
 msgid "TICKET_DESC_featured_artist_ticket"
-msgstr "We give our warm congratulations to you for becoming a Sudomemo Featured Artist!"
+msgstr "We geven je een warme felicitatie aan jou omdat je een uitgelichte artiest bent geworden!"
 
 msgid "TICKET_NAME_spookmaster_ticket"
 msgstr "Spookmaster Ticket"
 
+#game title not translated due to it being the name of a game which almost never gets translated
 msgid "TICKET_DESC_spookmaster_ticket"
-msgstr "Thank you so much for playing SudoQuest: Requiem! We hope you enjoyed the game."
+msgstr "We bedanken je om SudoQuest: Requiem! te spelen we hopen dat je de game leuk vond!"
 
 msgid "TICKET_NAME_nitro_booster_ticket"
 msgstr "Nitro Booster Ticket"
 
 msgid "TICKET_DESC_nitro_booster_ticket"
-msgstr "Hey, you! You're awesome! Thanks for boosting the Sudomemo Discord!"
+msgstr "He jij! Je bent geweldig! Bedankt voor het boosten van de sudomemo discord!"
 
 msgid "TICKET_NAME_7th_birthday_bash_ticket"
-msgstr "7th Birthday Bash Ticket"
+msgstr "7de Birthday Bash Ticket"
 
 msgid "TICKET_DESC_7th_birthday_bash_ticket"
-msgstr "Thanks for attending our Birthday Bash!"
+msgstr "Bedankt voor het meedoen met de 7de birthday bash!"
 
 msgid "TICKET_NAME_8th_birthday_bash_ticket"
-msgstr "8th Birthday Bash Ticket"
+msgstr "8ste Birthday Bash Ticket"
 
 msgid "TICKET_DESC_8th_birthday_bash_ticket"
-msgstr "Thanks for attending our Birthday Bash!"
+msgstr "bedankt voor het meedoen met de 8ste birthday bash!"
```

Comment:
The first letters of "Birthday" and "Bash" should be capitalized, and the number shouldn't be mentioned.